README.md

```diff
@@ -133,16 +133,35 @@ Ubuntu 16.04     | no
 Ubuntu 18.04     | yes
 
 
-## Localization
+## Localization and translation
```

Comment:
IMHO Localization is a better heading because localization included translation.

Reply:
I see no problem to point out translation because that is our focus. We do no other localization.